src/main/java/de/tum/in/www1/artemis/repository/StudentParticipationRepository.java

```diff
@@ -239,6 +239,7 @@ Optional<StudentParticipation> findByExerciseIdAndStudentIdAndTestRunWithEagerSu
                 LEFT JOIN FETCH p.results r
                 LEFT JOIN FETCH r.submission s
                 LEFT JOIN FETCH p.submissions
+                LEFT JOIN FETCH r.assessmentNote
```

Comment:
Do we actually need them every time we call this method?
If yes, please change the name of the method accordingly. If no, please write a new method

Reply:
@BaumiCoder I checked that the `api/exercises/{exerciseId}/participations` endpoint is only used in `participation.service.ts` `findAllParticipationsByExercise`. This in turn is only used by the `exercise-scores.component`.

=> only used by this one changed client component
=> Please add a `AndAssessmentNote` suffix to the method name for the two changed methods in this class.

Reply:
Suffix added with d3dd2d42802e980a91dfab53da45893a5b20f27c